src/flownet/ert/_flow_job.py

```diff
@@ -26,8 +27,27 @@ def run_flow():
     parser.add_argument("data_file", type=str)
     args = parser.parse_args()
 
-    subprocess.run(
-        [os.environ.get("FLOW_PATH", "/usr/bin/flow"), args.data_file], check=True
-    )
+    if "FLOW_PATH" in os.environ:
+        flow_path = os.environ.get("FLOW_PATH")
+        if os.path.isfile(flow_path):
+            # Runing flow from ENV variable
+            subprocess.run([flow_path, args.data_file], check=True)
+        else:
+            raise AssertionError("FLOW_PATH points to a path that doesn't exist")
+    elif shutil.which("flow") is None:
+        raise AssertionError(
+            "OPM/flow is not installed.\
+                             Follow instructions in https://opm-project.org/ to install flow."
+        )
```

Comment:
Mention here that _if_ it is already installed, the user should make sure it is available in `$PATH` (or alternatively use the environment variable `$FLOW_PATH`).